src/control/server/mgmt_svc.go

```diff
@@ -191,13 +191,15 @@ type mgmtSvc struct {
 	log        logging.Logger
 	harness    *IOServerHarness
 	membership *system.Membership // if MS leader, system membership list
+	cfg        *AgentCfg
```

Comment:
Suggest naming the field member agentConfig to make it clear later on what kind of config is being referenced.

Reply:
Done